### PR TITLE
Pricing page content updates

### PR DESCRIPTION
**Pull request by tdlowden**

### CODE REVIEW

**Comment by tdlowden** [comment]
Should we change the page name from pricing to something else?

File: app/templates/views/pricing/index.html [hunk 1]
```diff
@@ -28,29 +28,29 @@ <h1 class="heading-large">Pricing</h1>
 
   <p class="govuk-body">
     {% if not current_user.is_authenticated %}
-      <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.register') }}">Create an account</a> then add as many unique services as you need to.</p>
+      <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.register') }}">Create an account,</a> then add your unique service(s).</p>
```
Comment (rahearn) [inline app/templates/views/pricing/index.html:31]: [question] should we call these "services"? Or something more like "Program"?

Comment (tdlowden) [reply app/templates/views/pricing/index.html:31]: I left it as services for continuity throughout the rest of the content. I suppose we could go through and change all the times the word appears, but I think for now, `service` is fine.

Ideally, we do a lot more content overhaul once we have some staff devoted to it.

**Review by rahearn — APPROVED** [review]

File: app/templates/views/pricing/index.html [hunk 2]
```diff
@@ -28,29 +28,29 @@ <h1 class="heading-large">Pricing</h1>
...
     {% else %}
-      You can add as many unique services as you need to.
+      Additional unique services may be added, subject to approval by the U.S. Notify team.
```
Comment (stvnrlly) [inline app/templates/views/pricing/index.html:33]: What do we imagine for approval? There's no current mechanism for this, so is it just individual manual follow-up?

Comment (tdlowden) [reply app/templates/views/pricing/index.html:33]: I think even after we more concretely outline criteria, it feels to me like it will always be manual?

**Pull request merged**